docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst

```diff
@@ -1948,11 +1947,8 @@ it as two packages, ``helper`` and ``main``:
 Remove Retroactive Instances
 ----------------------------
 
-SCU eliminates the need for retroactive instances and is not
-compatible with them. To ensure package selection for interface choices
-acts correctly, retroactive interface instances should be moved to newer
-versions of templates, such that changes to the instance warrants a new
-version of the template.
+SCU is not compatible with retroactive instances, so these will need to be removed.
```

Comment:
In which situation would we "remove" them rather than not introduce them? I guess if you're compiling some old code to LF 1.17?

Reply:
Yeah, this is more in the context of migration, but migration points to this list. I'll see if theres better wording